js/charts/SlopeChartTransform.ts

```diff
@@ -124,37 +142,35 @@ export default class SlopeChartTransform implements IChartTransform {
     @computed get data(): SlopeChartSeries[] {
         if (!this.yDimension) return []
 
-        const { variableData, sizeDim, yDimension, xDomain, colorDim, colorScale, chart } = this
-        let data = variableData
+        const { yDimension, xDomain, colorByEntity, sizeByEntity, chart } = this
         const entityKey = this.chart.vardata.entityMetaByKey
 
-        // Make sure we're using time bounds that actually contain data
```

Comment:
note that this logic of calculating the longest range of years is not ported over. it is not needed looking at the charts on the test page. let me know if i am missing something here